src/nrnpython/CMakeLists.txt

```diff
@@ -87,33 +87,87 @@ else()
   set(BUILD_MEMACS_TRUE "#")
 endif()
 
-# =============================================================================
-# All variables are set, prepare setup.py for python module install
-# =============================================================================
-nrn_configure_file(setup.py src/nrnpython)
 
 # =============================================================================
-# Setup MinGW toolchain for setuptools
+# nrnpython libraries
 # =============================================================================
-if(MINGW)
-  file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/setup.cfg "[build]\ncompiler=mingw32")
-endif()
+add_library(rxdmath SHARED ${CMAKE_CURRENT_SOURCE_DIR}/rxdmath.c)
+install(TARGETS rxdmath DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
 
-# =============================================================================
-# Build rxdmath library
-# =============================================================================
-if(NRN_ENABLE_RX3D)
-  add_library(rxdmath SHARED ${CMAKE_CURRENT_SOURCE_DIR}/rxdmath.c)
-  install(TARGETS rxdmath DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
+
+# Created nrnpython
+set(nrnpython_lib_list)
+
+# user has selected dynamic python support (could be multiple versions)
+if(NRN_ENABLE_PYTHON_DYNAMIC)
+  set(INCLUDE_DIRS
+      .
+      ../oc
+      ../nrnoc
+      ../ivoc
+      ../nrniv
+      ../ivos
+      ../gnu
+      ../mesch
+      ../nrnmpi
+      ${PROJECT_BINARY_DIR}/src/nrnpython
+      ${PROJECT_BINARY_DIR}/src/ivos
+      ${PROJECT_BINARY_DIR}/src/oc
+      ${IV_INCLUDE_DIR})
+
+  if(LINK_AGAINST_PYTHON)
+    list(LENGTH NRN_PYTHON_EXE_LIST _num_pythons)
+    math(EXPR num_pythons "${_num_pythons} - 1")
```

Comment:
I'm curious, why -1 ?

Reply:
That logic wasn't changed, just moved from a code block below.